content/200-concepts/100-components/01-prisma-schema/08-views.mdx

```diff
@@ -253,7 +253,17 @@ view UserInfo {
 
 The `view` block is generated initially with a `@@ignore` attribute because [there is no unique identifier defined](#unique-identifier) (which is currently a [limitation](#unique-identifier) of the views preview feature).
 
-In this view's case, the `id` column refers to a uniquely identifiable field in the underlying `User` table so that field will also be used as the uniquely identifiable field in the `view` block.
+<Admonition type="warning">
+
+Please note for now `db pull` will only introspect views in your schema when using PostgreSQL. Support for this workflow will be extended to other database providers soon.
+
+</Admonition>
+
+#### Adding a unique identifier to an introspected view
+
+To be able to use the introspected view in Prisma Client, you will need to select and define one or multiple of the fields as the unique identifier.
```

Comment:
I would maybe update this to say "as a unique identifier" due to the multiple multiple

Reply:
Hmm grammar question. The "multiple of the fields" refers to `@@unique[field1, field2]`. Is it then more correct to say "a unique identifier"? I honestly don't know.

Reply:
ah, oh no, hmm, as a tech q, does @@unique([...]) refer to: individual fields not needing to be unique in and of themselves but the combination _does_ ? or is it just a set of unique identifiers

Reply:
The combination of the multiple fields need to be unique, yes. We sometimes call the "compound uniques".

If you have mutliple fields that need to be unique by themselves, then you use two `field1 Type @unique` and `field2 Type @unique` instead, so via attributes on the field directly.

Reply:
Ah, okay, in the case of `@@unique` then "the unique identifier" makes most sense to me